app/opportunities/[opportunity_id]/edit/_components/phases/phase.tsx

```diff
@@ -31,18 +33,48 @@ export default function Phase({ index, phase, remove: removePhase }: Props) {
     name: `phases.${index}.questionnaires`,
   });
 
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({
+      id:
+        phase.id ??
+        (() => {
+          // phase id can theoretically be optional
+          console.warn("No phase id found");
+          return "placeholder_id";
+        })(),
```

Comment:
@simon-hng I don't quite understand why phases could have an optional id. But when I tried removing the optional attribute from the zod definition, the application stopped working.
If you have a better solution for this lmk

Reply:
shouldn't be optional. Interesting but with removing the attribute. If you cannot find a quick way to resolve it create a new ticket in linear to document it